test/lib/jdk/test/lib/net/SimpleHttpServer.java

```diff
@@ -88,6 +89,10 @@ public String getAddress() {
         return _address;
     }
 
+    public int getPort() {
+        return _httpserver.getAddress().getPort();
+    }
+
```

Comment:
There are many issues with this class - using "localhost" and binding to the wildcard address among others. 
Having instance variables that are not final but are accessed by potentially multiple threads is another. 
I could also mention not using try-with-resources or the odd _name convention.
It will need to be modernized if you want to put it in jdk.test.lib.net;

Reply:
Thanks. Now this class is modernized in next patch.